shell/server/account-suspension.js

```diff
@@ -86,6 +86,10 @@ Meteor.methods({
       throw new Meteor.Error(403, "Only admins can suspend other users.");
     }
 
+    if (userId === Meteor.userId()) {
+      throw new Meteor.Error(400, "Admins cannot suspend their own accounts from the admin page. Please go to your account setttings.");
```

Comment:
Hilariously, users in an organization can't suspend their own accounts from the account settings page either.  But I guess having to ask another admin to suspend your account for you is probably okay in this circumstance.